src/common/managers/SceneManager.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+** EPITECH PROJECT, 2024
+** StellarForge
+** File description:
+** SceneManager.cpp
+*/
+
+#include "SceneManager.hpp"
+
+#include "ManagerException.hpp"
+
+void SceneManager::addScene(const UUID &uuid,
+                            const IScene *scene,
+                            int position /*= -1*/) {
```

Comment:
You already have put that in the description of the params
```suggestion
                            int position) {
```